library/ui-styles/src/main/res/values/theme_common.xml

```diff
@@ -25,4 +25,10 @@
         <item name="android:backgroundDimEnabled">false</item>
     </style>
 
+    <style name="Theme.Vector.ActionButton" parent="@android:style/Widget.ActionButton">
```

Comment:
This is not the correct location to add this new style. Can you move it to a new file `style_action_button.xml` in the same folder please?